src/mlpack/bindings/python/ConfigureSetup.cmake

```diff
@@ -1,46 +1,3 @@
 # ConfigureSetup.cmake: generate the setup.py file given several environment
 # variables.
-#
-# This file depends on the following variables being set:
-#
-# - SETUP_PY_IN: location of input file
-# - SETUP_PY_OUT: location of output file
-# - PACKAGE_VERSION: version of package
-# - Boost_SERIALIZATION_LIBRARY: location of Boost serialization library
-# - Boost_LIBRARY_DIRS: paths to boost libraries
-# - ARMADILLO_LIBRARIES: space-separated list of Armadillo dependencies
-# - MLPACK_LIBRARY: location of mlpack library
-# - MLPACK_PYXS: list of pyx files
-# - OpenMP_CXX_FLAGS: OpenMP C++ compilation flags
-# - DISABLE_CFLAGS: list of CFLAGS or CXXFLAGS to be disabled
-# - CYTHON_INCLUDE_DIRECTORIES: include directories for Cython
-# - MLPACK_LIBDIR: path to mlpack libraries
-# - OUTPUT_DIR: binary output directory for CMake
```

Comment:
We should probably keep the comment but remove the `Boost_SERIALIZATION_LIBRARY` line.  Anyway let's see if it works. :smile:

Reply:
Agreed.